olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml

```diff
@@ -1256,23 +1256,23 @@ spec:
       # This reference will be replaced in local builds and CI via hack/lib/catalogsource.bash.
       image: registry.ci.openshift.org/knative/openshift-serverless-nightly:knative-openshift-ingress
     - name: "IMAGE_queue-proxy"
-      image: "registry.ci.openshift.org/openshift/knative-v1.8.0:knative-serving-queue"
+      image: "registry.ci.openshift.org/openshift/knative-serving-queue:knative-v1.8"
     - name: "IMAGE_activator"
-      image: "registry.ci.openshift.org/openshift/knative-v1.8.0:knative-serving-activator"
+      image: "registry.ci.openshift.org/openshift/knative-serving-activator:knative-v1.8"
     - name: "IMAGE_autoscaler"
-      image: "registry.ci.openshift.org/openshift/knative-v1.8.0:knative-serving-autoscaler"
+      image: "registry.ci.openshift.org/openshift/knative-serving-autoscaler:knative-v1.8"
     - name: "IMAGE_autoscaler-hpa"
-      image: "registry.ci.openshift.org/openshift/knative-v1.8.0:knative-serving-autoscaler-hpa"
+      image: "registry.ci.openshift.org/openshift/knative-serving-autoscaler-hpa:knative-v1.8"
     - name: "IMAGE_controller__controller"
-      image: "registry.ci.openshift.org/openshift/knative-v1.8.0:knative-serving-controller"
+      image: "registry.ci.openshift.org/openshift/knative-serving-controller:knative-v1.8"
     - name: "IMAGE_webhook__webhook"
-      image: "registry.ci.openshift.org/openshift/knative-v1.8.0:knative-serving-webhook"
+      image: "registry.ci.openshift.org/openshift/knative-serving-webhook:knative-v1.8"
     - name: "IMAGE_domain-mapping"
-      image: "registry.ci.openshift.org/openshift/knative-v1.8.0:knative-serving-domain-mapping"
+      image: "registry.ci.openshift.org/openshift/knative-serving-domain-mapping:knative-v1.8"
     - name: "IMAGE_domainmapping-webhook"
-      image: "registry.ci.openshift.org/openshift/knative-v1.8.0:knative-serving-domain-mapping-webhook"
-    - name: "IMAGE_storage-version-migration-serving-serving-1.8.0__migrate"
-      image: "registry.ci.openshift.org/openshift/knative-v1.8.0:knative-serving-storage-version-migration"
+      image: "registry.ci.openshift.org/openshift/knative-serving-domain-mapping-webhook:knative-v1.8"
+    - name: "IMAGE_storage-version-migration-serving-serving-1.8__migrate"
```

Comment:
The name change seems unwanted? and it can cause a mismatch in the image replacement?

Reply:
Should work according to [discussion](https://redhat-internal.slack.com/archives/CF5ANN61F/p1681465574447549?thread_ts=1681450896.495659&cid=CF5ANN61F).